ote/src/cljs/ote/ui/buttons.cljs

```diff
@@ -51,3 +51,17 @@
              (stylefy/use-style style-buttons/svg-button)
              opts)
    label])
+
+(defn big-icon-button-with-label
```

Comment:
Just wondering, is there a need for a common component to "encapsulate" link or button inside same component?
Both have typically different handling so what's the advantage gained?

Reply:
Typically buttons are used when something needs to be happening on the same page. And links are used when user is moved to some other location. But in this case csv is downloaded from server and easiest way to do this is via link. But that link is rendered like a button.
Same thing is in own-services page, but without big icon.
Not sure if similar things are in other pages too.

Reply:
Indeed those are the use-cases for link and button elements, but
the question I wasn't sure about was if there's a need for component to encapsulate them, compared to one component which is a link but looks like a button and another component which is a button.
Probably I'm missing something.

Reply:
Component changed to button